ci/test.sh

```diff
@@ -63,6 +60,9 @@ echo "--- Testing Unity: Test Project Playmode :joystick:"
 pushd "test-project"
     dotnet run -p "${PROJECT_DIR}/.shared-ci/tools/RunUnity/RunUnity.csproj" -- \
         -batchmode \
+        -enableCodeCoverage \
+        -coverageResultsPath "${COVERAGE_RESULTS_PATH}/test-project" \
```

Comment:
Will it be oke that this and the editmode tests go to the same location?

Reply:
It creates an `Editmode/Playmode` folder for each 😄